src/clj/puppetlabs/services/jruby/jruby_puppet_schemas.clj

```diff
@@ -28,6 +28,9 @@
     * :track-lookups - a boolean to turn on tracking hiera lookups during compilation;
         if not specified, no tracking is enabled.
 
+    * :disable_i18n - a boolean to pass to Puppet to control whether or not to translate;
+        if not specified, uses Puppet's own default.
```

Comment:
It looks like above we default to the value of "multithreaded", meaning this key will always be present and will always override Puppet's default. Or did I misunderstand what's going on in `initialize-puppet-config`? I think that behavior is probably correct, but that makes this comment misleading.

Reply:
Oh, that should be a hyphen; if `disable-i18n` is not in the jruby-puppet config, then it is not processed in `config->puppet-config`, and thus uses whatever value Puppet defaults to for `disable_i18n`.

Reply:
But will it ever not be in there? This line https://github.com/puppetlabs/puppetserver/pull/2222/files/ffef18266775488e80039179603feea8e714b374#diff-5ad97bf28dd9f8f97bab3706b0367684R205 should set it to `multithreaded` if it is either `false` or `nil`, I think...